src/components/m-header-languages/_template.js

```diff
@@ -2,18 +2,31 @@ import html from 'nanohtml';
 import raw from 'nanohtml/raw';
 import classnames from 'classnames';
 
-export default ({ items }) => [html`
+const getSelectedName = (value, items) => {
+  if (!items || !Array.isArray(items) || !items.length) {
+    return '';
+  }
+  const selectedItem = items.find(item => item.name === value);
```

Comment:
Very good the usage of find, but please add a polyfill or consider using filter instead as we still have to support ie11